### PR TITLE
Fix Google Drive Webhooks renewer

### PR DESCRIPTION
**Pull request by lasryaric**

### CODE REVIEW

File: connectors/src/connectors/google_drive/temporal/activities.ts [hunk 1]
```diff
@@ -592,32 +595,52 @@ export async function renewWebhooks(pageSize: number) {
...
       } else {
         await wh.update({
           webhookId: webhookInfo.value.id,
           expiresAt: new Date(webhookInfo.value.expirationTsMs),
         });
       }
+    } catch (e) {
+      logger.error(
+        { error: e },
+        `Failed to renew webhook for ${wh.connectorId}`
```
Comment (spolu) [inline connectors/src/connectors/google_drive/temporal/activities.ts:628]: logs should not be templated (to simplify search) wh.connectorId should be in the object above:
{ error: e, connectorId: wh.connectorId }

I would also add the workspaceId here

File: connectors/src/connectors/google_drive/temporal/activities.ts [hunk 2]
```diff
@@ -592,32 +595,52 @@ export async function renewWebhooks(pageSize: number) {
...
+      );
+      const tags = [
+        `connector_id:${wh.connectorId}`,
+        `workspaceId:${connector.workspaceId}`,
+      ];
+      statsDClient.increment(
+        "`google_drive_renew_webhook_errors.count",
```
Comment (spolu) [inline connectors/src/connectors/google_drive/temporal/activities.ts:635]: extra `

**Review by spolu — CHANGES_REQUESTED** [review]
2 small changes, feel free to merge directly once addressed

File: connectors/src/connectors/google_drive/temporal/activities.ts (hunk 1 above)
Comment (lasryaric) [reply connectors/src/connectors/google_drive/temporal/activities.ts:628]: My bad

File: connectors/src/connectors/google_drive/temporal/activities.ts (hunk 2 above)
Comment (lasryaric) [reply connectors/src/connectors/google_drive/temporal/activities.ts:635]: thanks.

**Pull request merged**